rust-gpu-toy/README.md

```diff
@@ -0,0 +1,12 @@
+# Compute shader toy based on wgpu
+
+This is a starting point for running compute shaders and drawing the image output into a window, based on [wgpu]. You should be able to change the shader (paint.wgsl) and run using simply `cargo run`.
```

Comment:
This looks like the wgpu one copy-pasted; I imagine there might be some significant updates for rust-gpu.

Reply:
Yeah, the first commit is copying and pasting the wgpu example, ideally to make reviewing easier. It should work the same way. I've been doing some work accross the ecosystem based on my learnings from this (e.g. https://github.com/Lokathor/bytemuck/pull/67 and https://github.com/EmbarkStudios/rust-gpu/pull/663)

I also believe the wgpu 0.7 being required was actually because of the pipeline inference, so 0.8 should be fine. I'll re-upgrade it soon.

Reply:
Oh, because it's getting the layout from `pipeline.get_bind_group_layout(0)` rather than building it explicitly? kvark mentioned something about this but I didn't realize it was actually the source of the problems. My understanding is that the wgpu compute example is that way to give coverage to pipeline inference, rather than the idea that it's a model for the way things should be done. In any case, this is good to know.